Ginger/Ginger/Actions/ActionConversion/SelectBusinessFlowWzardPage.xaml.cs

```diff
@@ -0,0 +1,188 @@
+                    break;
+                case EventType.LeavingForNextPage:
+                    SetActivitiesSelected();
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// This method is used to set the acitvities for selected to conversion
+        /// </summary>
+        private void SetActivitiesSelected()
+        {
+            foreach (var businessFlow in ListOfBusinessFlow)
+            {
+                if (businessFlow.Selected)
+                {
+                    foreach (var act in businessFlow.Activities)
+                    {
+                        act.SelectedForConversion = true;
+                    } 
+                }
+            }
+        }
+
+        /// <summary>
+        /// This method sets the gridview to display the selected businessflows
+        /// </summary>
+        private void SetGridsView()
+        {
+            GridViewDef defView = new GridViewDef(GridViewDef.DefaultViewName);
+            defView.GridColsView = new ObservableList<GridColView>();
+            defView.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlow.Selected), WidthWeight = 5, MaxWidth = 50, StyleType = GridColView.eGridColStyleType.CheckBox, Header = "Select", BindingMode = System.Windows.Data.BindingMode.TwoWay });
+            defView.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlow.ContainingFolder), WidthWeight = 15, Header = "Folder" });
+            defView.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlow.Name), WidthWeight = 15, Header = "Name" });
+            defView.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlow.Description), WidthWeight = 15, Header = "Description"  });
+                        
+            xBusinessFlowGrid.SetAllColumnsDefaultView(defView);
+            xBusinessFlowGrid.InitViewItems();
+            xBusinessFlowGrid.SetTitleLightStyle = true;
+            xBusinessFlowGrid.btnMarkAll.Visibility = System.Windows.Visibility.Visible;
+            xBusinessFlowGrid.btnAdd.AddHandler(Button.ClickEvent, new RoutedEventHandler(AddBusinessFlow));
+
+            xBusinessFlowGrid.DataSourceList = SetDefaultSelectedBusinessFlows();
+            xBusinessFlowGrid.RowChangedEvent += grdGroups_RowChangedEvent;
+            xBusinessFlowGrid.Title = GingerDicser.GetTermResValue(eTermResKey.BusinessFlows) + " to Convert";
+            xBusinessFlowGrid.MarkUnMarkAllActive += MarkUnMarkAllActivities;
+            xBusinessFlowGrid.ValidationRules = new List<ucGrid.eUcGridValidationRules>()
+            {
+                ucGrid.eUcGridValidationRules.CheckedRowCount
+            };
+        }
+        
+        /// <summary>
+        /// This method by default selects the businessflows to convert
+        /// </summary>
+        /// <returns></returns>
+        private IObservableList SetDefaultSelectedBusinessFlows()
+        {
+            foreach (BusinessFlow bf in ListOfBusinessFlow)
+            {
```

Comment:
also here, why bf.Selected is been used?

Reply:
as above removed